python/nano/src/bigdl/nano/quantization/quantization_inc.py

```diff
@@ -0,0 +1,83 @@
+from neural_compressor.conf.config import Quantization_Conf
+from neural_compressor.experimental import Quantization
+
+
+class QuantizationINC(Quantization):
+    def __init__(self,
+                 framework,
+                 conf=None,
+                 approach='ptsq',
```

Comment:
The possible parameter you passed are:
```python
            approach_map = {
                'static': 'post_training_static_quant',
                'dynamic': 'post_training_dynamic_quant'
            }
```
Maybe use the full name to be clearer?

Reply:
How about rename `trainer.quantize` as `trainer.post_training_quantize(...)`?  Is it much more convenient for users to call? 'post_training_static_quant' is a bit too long, sometimes I need to check the document for valid options.

Reply:
maybe use enum or constant variable. I actually prefer just `trainer.quantize`